web-ng/src/app/shared/models/project.model.ts

```diff
@@ -15,10 +15,16 @@
  */
 
 import { StringDictionary } from "./string-dictionary.model";
+import { Layer } from "./layer.model";
+
+type LayerMap = {
+  [key: string]: Layer;
+}
 
 export class Project {
   constructor(
     readonly title: StringDictionary,
-    readonly description: StringDictionary
-  ) {}
+    readonly description: StringDictionary,
+    readonly layers: LayerMap,
```

Comment:
In `master` I have this represented as an `Immutable.List<Layer>`. If it's more convenient to leave it as a map we can do that as well. If so, when merging please replace this with an `Immutable.Map` instead.

Reply:
List sounds good:)